pkg/workflows/create.go

```diff
@@ -99,6 +99,14 @@ type InstallCuratedPackagesTask struct{}
 
 func (s *CreateBootStrapClusterTask) Run(ctx context.Context, commandContext *task.CommandContext) task.Task {
 	if commandContext.BootstrapCluster != nil {
+		if commandContext.ClusterSpec.AWSIamConfig != nil {
+			logger.Info("Creating aws-iam-authenticator certificate and key pair secret on bootstrap cluster")
+			if err := commandContext.ClusterManager.CreateAwsIamAuthCaSecret(ctx, commandContext.BootstrapCluster, commandContext.ClusterSpec.Cluster.Name); err != nil {
+				commandContext.SetError(err)
+				return &CollectMgmtClusterDiagnosticsTask{}
+			}
```

Comment:
This is being called here as well as on line 141 below. Can you move this before `if commandContext.BootstrapCluster != nil {` check on line 101 and remove it on line 141?

Reply:
Actually I think that might not work? The call on line 141 actually uses the newly created KinD bootstrap cluster, and that reference is only available on 120. That is why I had to repeat the calls.